src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -249,14 +262,85 @@ function genDockerFile(serviceNameAndRelativePath: string, platform: Platform, o
         .replace(/\$assembly_name\$/g, assemblyNameNoExtension)
         .replace(/\$copy_project_commands\$/g, copyProjectCommands);
 
+    validateForUnresolvedToken(contents);
+
+    return contents;
+}
+
+function validateForUnresolvedToken(contents: string): void {
     let unreplacedToken = extractRegExGroups(contents, /(\$[a-z_]+\$)/, ['']);
     if (unreplacedToken[0]) {
         assert.fail(`Unreplaced template token "${unreplacedToken}"`);
     }
+}
 
-    return contents;
+function generateComposeFiles(dockerfileName: string, platform: Platform, os: PlatformOS | undefined, ports: number[], artifactName: string): ScaffoldFile[] {
+    const serviceName = path.basename(artifactName, path.extname(artifactName)).toLowerCase();
+    const imageName = serviceName;
+    let jsonPorts: string = '';
+    if (ports && ports.length > 0) {
+        const portsArray = ports.map(p => `      - ${p}`).join("\r\n");
+        jsonPorts = `
+    ports:
+${portsArray}`;
+    }
+
+    let environmentVariables: string = '';
+    if (platform === 'ASP.NET Core') {
```

Comment:
Hani's code might change this `ASP.NET Core` value, make sure these get merged right.

Reply:
yes.